Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
That being said, you mentioned to me that there was a problem accessing type objects in SCOOP mode. There might be some other issues in multi-threaded mode as well. Does that mean that we would be better off if this class was expanded? In that case, it would not be that useful anymore to keep the computed name objects in an internal attribute as we do now. So these objects would be very small.